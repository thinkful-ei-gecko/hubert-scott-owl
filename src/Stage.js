import React from 'react';

export default function Stage(props) {
    return (
        <div className="user-info" key={props.id}>
            <div className="user-name">
                {props.name}
            </div>
            <div className="volume-line"></div>
            <i className="fas fa-volume-up"></i>
            <i className="fas fa-search"></i>
            <div className="user-avatar">
                <img src={props.avatar} alt ="user avatar"/>
            </div>
        </div>
    )
}