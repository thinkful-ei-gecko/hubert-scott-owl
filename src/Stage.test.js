import React from 'react';
import ReactDOM from 'react-dom';
import Stage from './Stage';
import renderer from 'react-test-renderer';
import { exportAllDeclaration } from '@babel/types';

describe('<Stage />', () => {
    it('renders without crashing', ()=> {
        const div = document.createElement('div');

        ReactDOM.render(<Stage id = 'a' name = 'frank' avatar = 'https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1'/>, div)
    
        ReactDOM.unmountComponentAtNode(div);
    });

});

describe('<Stage />', () => {
    it('renders UI correctly', () => {
        const tree = renderer.create(<Stage id = 'a' name = 'frank' avatar = 'https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1'/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});