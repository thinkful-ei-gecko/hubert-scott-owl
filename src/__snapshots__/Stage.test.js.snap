// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Stage /> renders UI correctly 1`] = `
<div
  className="user-info"
>
  <div
    className="user-name"
  >
    frank
  </div>
  <div
    className="volume-line"
  />
  <i
    className="fas fa-volume-up"
  />
  <i
    className="fas fa-search"
  />
  <div
    className="user-avatar"
  >
    <img
      alt="user avatar"
      src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
    />
  </div>
</div>
`;
